### ISSUE TITLE
Proton Pass: how do I delete all my passwords?

### ISSUE THREAD

**Issue by Deuchnord** [ProtonMail/WebClients#356]
> We are happy to answer your questions about the code or discuss technical ideas.

Please complete the following checklist (by adding [x]):

-   [x] I have searched open and closed issues for duplicates
-   [ ] This isn't a feature request
-   [x] This is not a report about my app not working as expected

---

Hello

Last month, I have imported all my passwords from Bitwarden into Proton Pass and used it on a daily base, but after reflection, I have chosen to get back to Bitwarden (just a personal preference, don't worry, your passwords manager is great too :wink:).
I don't plan to use Proton Pass anymore, so I'd like to delete all my passwords for security, but I can't find any option for that. How do I do this?

**Comment by halwel** [comment]
I realize this is probably too late to help you, Deuchnord, but I just needed to do the same thing after doing an import from another app and figured out a way to do it, so I decided to post it here for anyone else who looks up this question.  I wanted to redo the import, so I wanted to delete the first import completely.  I found a way to do it pretty easily.  What I did was, after noticing there’s a Delete Vault option, but finding it doesn’t work on the main, Personal vault…and also noticing there’s a “Move All Items” option…I first made a new Vault called Temp; then, when you pull down the menu for/next-to the main vault (Personal) (the three dots menu) one of the options is “Move all Items,” so I chose that and selected the Temp vault I’d just made as the destination, and everything moved to that new Temp vault, leaving Personal empty.  Then I deleted the Temp vault, and everything was gone—didn’t even put it in the Trash, it just completely wiped everything.  Kind of a work-around, but it got the job done.  

Hope this helps anyone else who looks up this question in the future. :)

**Comment by Deuchnord** [comment]
It's not too late for me, I'm still looking for a way to delete all my passwords. My vault contains hundreds of passwords, deleting them manually would take too much time.

**Comment by szamanr** [comment]
i run into the same in my situation. i had imported my vault from bitwarden to try proton pass out, then i kept using bitwarden and accumulated new passwords. now i wanted to sync my proton pass vault. unfortunately, a new import resulted in all entires being duplicated.

as a workaround for that issue, i wanted to remove all my passwords and import again, but this is impossible.

so, a workaround for that:

1. create a new vault called e.g. "tmp"
2. move all items to the new vault (using the menu next to your vault's name)
3. delete the new vault, which will also delete all items

hope this helps someone.

PS. i just realised the post above suggests exactly the same. i hope mine is easier to read at least. :)

**Comment by sd1s** [comment]
Thank you for publishing your findings - It has helped me a lot. i was going mad.
2025 update. 
1. Created tmp Vault.
2. Select items to move to tmp vault as no longer an option ‘Move all Items’
3. move to and select the vault you would like to move the items to.
4. Done.
5. It may look like the items havent moved but it is displaying ALL items across ALL Vaults. so its just displaying what’s in the Tmp Vault
6. Delete the tmp vault.

Basically what they said above but it’s change a little.
